src/announcement/AnnouncementController.ts

```diff
@@ -47,14 +47,14 @@ const defaultState = {
 };
 
 /**
- * Controller for managing in-app announcement notifications.
+ * Controller for managing in-app announcements.
  */
-export class NotificationController extends BaseController<
+export class AnnouncementController extends BaseController<
   NotificationConfig,
```

Comment:
Could we rename these as well? These are public exports, so if we don't change them now, it will require another breaking change

Reply:
Done!